services/apps/alcs/src/alcs/application/application-submission/application-submission.service.ts

```diff
@@ -82,4 +85,25 @@ export class ApplicationSubmissionService {
       statusCode,
     );
   }
+
+  async update(
+    fileNumber: string,
+    updateDto: AlcsApplicationSubmissionUpdateDto,
+  ) {
+    //Load submission without relations to prevent save from crazy cascading
+    const submission = await this.applicationSubmissionRepository.findOneOrFail(
```

Comment:
a nitpick: this could be moved to a separate function and reused in service

Reply:
done!